Train.py

```diff
@@ -24,63 +29,77 @@
     save_data = torch.load(args.resume_path)
     net.load_state_dict(save_data)
 
-loss_record = {}
-loss_record['train'] = Utils.Loss_Record()
-loss_record['val'] = Utils.Loss_Record()
-
-rate_counter = Utils.Rate_Counter()
-
 data = Data.Data()
-
-timer = {}
-timer['train'] = Timer(args.mini_train_time)
-timer['val'] = Timer(args.mini_val_time)
-print_timer = Timer(args.print_time)
-save_timer = Timer(args.save_time)
+batch = Batch.Batch(net)
 
 # Maitains a list of all inputs to the network, and the loss and outputs for
 # each of these runs. This can be used to sort the data by highest loss and
 # visualize, to do so run:
 # display_sort_trial_loss(data_moment_loss_record , data)
 data_moment_loss_record = {}
+rate_counter = Utils.Rate_Counter()
 
-batch = Batch.Batch(net)
-
-while True:
-    for mode, data_index in [('train', data.train_index),
-                             ('val', data.val_index)]:
-        timer[mode].reset()
-        while not timer[mode].check():
-
-            batch.fill(data, data_index)  # Get batches ready
+def run_net(data_index):
+    batch.fill(data, data_index)  # Get batches ready
+    batch.forward(optimizer, criterion, data_moment_loss_record)
 
-            # Run net, forward pass
-            batch.forward(optimizer, criterion, data_moment_loss_record)
+try:
+    for epoch in range(1000):
```

Comment:
This should be a parameter or at least a constant defined at the top of the file.